project1/application.py

```diff
@@ -90,4 +90,22 @@ def authenticate():
 @app.route("/logout")
 def logout():
     session.clear()
-    return render_template("registration.html")
\ No newline at end of file
+    return render_template("registration.html")
+
+@app.route('/review', methods =['GET','POST'])
+def review():
+    if request.method == 'POST':
+        rating = request.form.get('review_tags')
+        review = request.form.get('review_value')
+        email = 'satish'
+
+        if (REVIEW.query.filter_by(email = email, isbn = "1234").first() == None) :
+            data = REVIEW(email = email, isbn = "1234", rating = rating, review = review) 
+            db1.session.add(data)
+            db1.session.commit()
+        else:
+            total_reviews = db1.session.query(REVIEW).filter(REVIEW.isbn == "1234")
```

Comment:
When does it goes into else condition can you explain.

Reply:
in else condition, all the reviews that are in the isbn number will get displayed.
for ex: nikhil gave review as good for isbn =1234
satish gave review as average for isbn = 1234
it will store in total_reviews

Reply:
You will not get the sum of the reviews.For your example,you will get total_reviews as what nikhil gave.Trace it you will get the answer or discuss with your team mates and update.

Reply:
Yes ma'am, that's correct. If i change my email from nikhil to satish then, both the reviews will be displayed.

Reply:
do you mean total_reviews is sum of reviews for a particular isbn?

Reply:
Complete details will be displayed ma'am.